### DISCUSSION TITLE
Cant install coqui-tts=0.24.2

### DISCUSSION THREAD

**Discussion by sulfur17** [DrewThomasson/ebook2audiobook#41] [Q&A]
on installation step 5 (Install Python packages) I input `pip install coqui-tts==0.24.2 pydub nltk beautifulsoup4 ebooklib tqdm gradio==4.44.0` in PowerShell and get error:
```
ERROR: Ignored the following versions that require a different python version: 0.22.1 Requires-Python <3.12,>=3.9.0; 0.23.0 Requires-Python <3.12,>=3.9.0; 0.23.1 Requires-Python <3.13,>=3.9.0; 0.24.0 Requires-Python <3.13,>=3.9; 0.24.1 Requires-Python <3.13,>=3.9; 0.24.2 Requires-Python <3.13,>=3.9; 0.24.3 Requires-Python <3.13,>=3.9
ERROR: Could not find a version that satisfies the requirement coqui-tts==0.24.2 (from versions: none)
ERROR: No matching distribution found for coqui-tts==0.24.2
```
what can I do? Help please

**Comment by sulfur17** [comment]
looks like I installed Python 3.13 instead of 3.10.
Now I setup 3.10 and retry step 5, but got same error :(

**Comment by ROBERT-MCDOWELL** [comment]
wait our version 2.0.0 so you won't have to do anything but run a windows/mac/linux script

**Comment by DrewThomasson** [comment]
## While you wait the easiest solution is to use the Docker lol

### Install and launch docker on your system:
https://www.docker.com/products/docker-desktop/

### And then Launch it with the docker run command:
### 🐳 Using Docker

You can also use Docker to run the eBook to Audiobook converter. This method ensures consistency across different environments and simplifies setup.

#### 🚀 Running the Docker Container

To run the Docker container and start the Gradio interface, use the following command:

 -Run with CPU only
```powershell
docker run -it --rm -p 7860:7860 --platform=linux/amd64 athomasson2/ebook2audiobookxtts:huggingface python app.py
```
 -Run with GPU Speedup (Nvida graphics cards only)
```powershell
docker run -it --rm --gpus all -p 7860:7860 --platform=linux/amd64 athomasson2/ebook2audiobookxtts:huggingface python app.py
```

This command will start the Gradio interface on port 7860.(localhost:7860)

#### Or try using miniconda to create your python 3.10 env , it's much easier to manage them with and allows you to install multiple version of python and switch between them easily.
- Idk it just sounds like your pip install isn't installing the correct python env is why

**Reply by sulfur17** [reply]
sounds complicated to me, better try to wait for version 2.0.0 XD
